.github/workflows/ci.yml

```diff
@@ -22,3 +26,14 @@ jobs:
         run: cargo build --verbose
       - name: Test the code
         run: cargo test --verbose
+  check-formatting:
```

Comment:
[This run](https://github.com/dguo/strsim-rs/actions/runs/7411172671/job/20165122755?pr=65) shows an example of a failure.